labs/open-enterprise-agent.md

```diff
@@ -0,0 +1,98 @@
+
+## Current Features
+
+### PLATFORM SUPPORT
+
+- Server
+- Kubernetes
+- Docker
+
+### AGENT TYPES
+
+- Issuer
+- Holder
+- Verifier
+
+### CREDENTIAL TYPES SUPPORTED
+
+- W3C JWT-VC
+- Anoncreds
+
+### DID METHODS SUPPORTED
+
+- did:prism
+- did:peer
+- Other did-methods can be verified via a universal resolver
+
+### COMMUNICATION SUPPORT
+
+- DIDCOMM V2
+
+### DIDCOMM PROTOCOLS SUPPORTED
+
+- [Mediator Coordinator](https://didcomm.org/mediator-coordination/2.0/)
+- [DIDComm V2 Messaging](https://identity.foundation/didcomm-messaging/spec)
+- [DIDComm V2 Issue Credential](https://github.com/decentralized-identity/waci-didcomm/tree/main/issue_credential)
+- [DIDComm V2 Present Proof](https://github.com/decentralized-identity/waci-didcomm/blob/main/present_proof/present-proof-v3.md)
+- [DIDComm V2 Report Problem](https://identity.foundation/didcomm-messaging/spec/#problem-reports)
+- [DIDComm V2 Routing Protocol](https://identity.foundation/didcomm-messaging/spec/#routing-protocol-20)
+
+### SUPPORTED ARIES RFC’s
+
+- [0023-did-exchange](https://github.com/hyperledger/aries-rfcs/tree/main/features/0023-did-exchange)
+- [0434-out-of-band-protocol](https://github.com/hyperledger/aries-rfcs/blob/main/features/0434-outofband/README.md)
+- [0453-issue-credential-protocol](https://github.com/hyperledger/aries-rfcs/tree/main/features/0453-issue-credential-v2)
+- [0453-present-proof-protocol](https://github.com/hyperledger/aries-rfcs/tree/main/features/0454-present-proof-v2)
+
+# Initial Committers
+- https://github.com/patlo-iog
+- https://github.com/davidpoltorak-io
+- https://github.com/antonbaliasnikov
+- https://github.com/FabioPinheiro
+- https://github.com/mineme0110
+- https://github.com/bvoiturier
+- https://github.com/yshyn-iohk
+- https://github.com/CryptoKnightIOG
+- https://github.com/shotexa
+- https://github.com/goncalo-frade-iohk
+- https://github.com/lohanspies
+
+# Initial Maintainers
+
+- https://github.com/bsandmann
+- https://github.com/patlo-iog
+- https://github.com/davidpoltorak-io
+- https://github.com/antonbaliasnikov
+- https://github.com/yshyn-iohk
+
+# Sponsors
+- Lance Byrd - ([lance.byrd@rootsid.com](mailto:lance.byrd@rootsid.com)) - Aries DIDCommV2 WG member
+- Stephen Curran (swcurran@cloudcompass.ca) - ACA-Py Maintainer, Aries RFC Maintainer
+- Timo Glastra - ([timo@animo.id](mailto:timo@animo.id)) - Aries Framework JavaScript Maintainer, ACA-Py Contributor
+- Daniel Hardman - ([daniel.hardman@gmail.com](mailto:daniel.hardman@gmail.com)) - Lead of ToIP Trust Spanning Task Force, Primary editor ToIP ACDC Task Force, Aries RFC Maintainer
+- Darrell O’Donnell - ([darrell.odonnell@continuumloop.com](mailto:darrell.odonnell@continuumloop.com)) - Chair of ToIP Technology Stack WG, Chair of ToIP Technology Architecture Task Force & Lead of ToIP Trust Registry Task Force
+
+# Pre-existing repository
+- https://github.com/input-output-hk/atala-prism-building-blocks
```

Comment:
I could not look at this repo. Is it public? I would like to understand the license and ensure DCO sign-off of all commits in this repo, which is a standard step that I prior to approving a lab.

Reply:
It's not public at present, I will work with our team to make that happen if needed.

Reply:
I will work with the team to ensure you get access. Given it's near nighttime in our timezone, I hope we can get that sorted tomorrow morning.

Reply:
@tkuhrt you should have access to the repository. Please let me know if there's anything else you need.

Reply:
I do have access to the repository. This leads to the following questions:

1. I am confused by the LICENSES.md file. It appears that there is a separate license listed for each of the dependencies. What is the LICENSE for the source code that you are looking to bring into Hyperledger?
2. The code does not have DCO sign-off, which is required for code contributed to Hyperledger Labs. There are two ways that we can resolve this. (1) Retroactively gather DCO sign-offs for the existing commits (2) Create a new repo and commit the code without the history, but using DCO sign-off. Which approach do you plan to take with this repo?

Reply:
Hi @tkuhrt !

I would like to clarify about the DCO requirement:
> The code does not have DCO sign-off, which is required for code contributed to Hyperledger Labs

For Atala PRISM, we're using [commits signature verification](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) and we sign each commit with our PGP keys that we officially publish and associate with an e-mail of each contributor.

We have a requirement that every commit must be signed and verified with a contributor PGP key.

Does the DCO requirement to put an additional string `Signed-off-by ...` to the body of each commit apply in this case? Could you, please, clarify if it is still required if the commit has already passed a signature verification?

Thank you!

Reply:
We will be working on seeing if we can resign to meet the dco requirement. We expect results to be clear next week. If that doesn't work, we'll likely squash as we move over.

Reply:
We've retroactively DCO signed the repo: https://github.com/input-output-hk/atala-prism-building-blocks/compare/a271709a789fd28d9404d5e0660b0d84bce41d1b...3c38dae89e8852ee181b69aa06910909d654a447.